src/styles/blocks/header.scss

```diff
@@ -0,0 +1,83 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  align-items: center;
+  outline: none;
```

Comment:
The `outline` property is generally used for accessibility purposes, indicating focus state of elements. If it's being set to 'none' without providing an alternative visual focus indicator, it could harm the accessibility of the website. Please ensure that there is an alternative focus style if you are going to remove the default outline.